geomstats/_backend/pytorch/__init__.py

```diff
@@ -39,6 +39,7 @@
     lt as less,
     matmul,
     max as amax,
+    mean,
```

Comment:
Since numpy's and torch's API of mean are different (`axis` is `dim` in torch) we should keep the lines below. Or use https://github.com/geomstats/geomstats/pull/540

If you are interested in using this, let me know and I'll merge the PR.